validator/src/main/java/com/google/daq/mqtt/registrar/LocalDevice.java

```diff
@@ -112,12 +113,17 @@
   );
 
   private static final Set<String> OPTIONAL_FILES = ImmutableSet.of(
-      RSA2_PUBLIC_PEM, RSA3_PUBLIC_PEM, ES2_PUBLIC_PEM, ES3_PUBLIC_PEM,
+      RSA_PUBLIC_PEM, RSA2_PUBLIC_PEM, RSA3_PUBLIC_PEM, 
```

Comment:
RSA_ and ES_ shouldn't be optional.  They are strictly required if that's the designated key type. If there's an error you're encountering then those cases should be handled more explicitly.   (E.g., RSA_PUBLIC_PEM is required iff the auth type is RSA.)

Reply:
OK, understood. I've changed the code to be specific about the RSA and ES keys and certs.